lib/utils/pyexec.c

```diff
@@ -470,6 +479,7 @@ STATIC int pyexec_friendly_repl_process_char(int c) {
         } else if (ret == CHAR_CTRL_E) {
             // paste mode
             serial_write_compressed(translate("\npaste mode; Ctrl-C to cancel, Ctrl-D to finish\n=== "));
+            echo(CHAR_CTRL_E);
```

Comment:
This character is defined as "enquiry". Some terminal programs respond to it, for instance the `rxvt` terminal on Linux will respond with the binary string `\033[1?2c`, explained as
> "ESC [ ? 1 ; 2 c" (``I am a VT100 with Advanced Video Option'')

The other characters you used have some kind of meaning in ASCII (and, by inheritance, as unicode/utf-8) but ^E is the one that I think may cause a problem with "real, in the world" terminals.